gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.0.1-all.zip
```

Comment:
Could you please execute `./gradlew wrapper` to regenerate the stuff in the [`gradle/wrapper` directory](https://github.com/JetBrains/rd/tree/master/gradle/wrapper), too?

Reply:
How could I forget about that? Actually it's relevant at the moment